ouroboros-network/src/Ouroboros/Network/TxSubmission/Inbound.hs

```diff
@@ -6,6 +6,8 @@
 {-# LANGUAGE NamedFieldPuns      #-}
 {-# LANGUAGE ScopedTypeVariables #-}
 
+{-# OPTIONS_GHC -Wno-partial-fields #-}
```

Comment:
We avoid disabling this one; I'll comment again near the use case.

Reply:
Removed!